css/fieldmanager.css

```diff
@@ -369,3 +369,22 @@ a.fm-delete:hover {
 .form-field .fm-checkbox label {
 	display: inline;
 }
+
+/* character counter */
+.fm-character-counter {
+	position: relative;
+	display: inline;
+}
+
+.fm-character-counter span {
+	position: absolute;
+	right: 10px;
+	top: -1px;
+	opacity: 1;
+	color: #aaa;
+	font-size: 0.9rem;
+}
+
+.fm-character-counter-padding {
+	padding-right: 35px !important;
```

Comment:
As I wrote, even if the initial PR's intent was to add counter to FM fields, the `fm_add_character_counter()` function could add the counter also to all WP admin fields.
The `!important` clause allows to override the WP admin style without need to add the complete selectors for all IDs. For example for the title post we would have to add:

```css
#titlediv #title.fm-character-counter-padding {
	padding-right: 35px ;
}
```